app/models/DepartureStatus.scala

```diff
@@ -50,11 +50,12 @@ object DepartureStatus extends Enumerable.Implicits with MongoDateTimeFormats {
 
   case object MrnAllocated extends DepartureStatus {
     override def transition(messageReceived: MessageReceivedEvent): DepartureStatus = messageReceived match {
-      case MessageReceivedEvent.MrnAllocated                => MrnAllocated
-      case MessageReceivedEvent.ControlDecisionNotification => ControlDecisionNotification
-      case MessageReceivedEvent.NoReleaseForTransit         => NoReleaseForTransit
-      case MessageReceivedEvent.ReleaseForTransit           => ReleaseForTransit
-      case _                                                => throw new Exception(s"Tried to transition from MrnAllocated to $messageReceived.")
+      case MessageReceivedEvent.MrnAllocated                   => MrnAllocated
+      case MessageReceivedEvent.ControlDecisionNotification    => ControlDecisionNotification
+      case MessageReceivedEvent.NoReleaseForTransit            => NoReleaseForTransit
+      case MessageReceivedEvent.ReleaseForTransit              => ReleaseForTransit
+      case MessageReceivedEvent.DeclarationCancellationRequest => DeclarationCancellationRequest
+      case _                                                   => throw new Exception(s"Tried to transition from MrnAllocated to $messageReceived.")
```

Comment:
Will this message be returned to the 3rd party app?  If so, it could do with some Matthew loving.

Reply:
In this case no, as I've just realised we're not using the transition on the public side. Do we want to ensure we are?

Reply:
I'm updating the code to at least force it to use the transition here, that way we at least get the exception. If we complete the ticket John put together for using an Either, we will see the message, at the moment we don't.